packages/frontend-2/components/global/AvatarGroup.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="flex items-center">
```

Comment:
We can merge these in `frontend-2` as well, but we should remember to clean up/delete our mock/test/fake components once we don't need them anymore